README.md

```diff
@@ -32,7 +39,24 @@ The following ways to begin a commit message are special and always valid:
 
 Another special scenario is to do `git commit --no-verify` which will skip the commit-msg hook and bypass commitplease.
 
-Common commit messages follow one of the style guides ([jQuery Commit Guidelines][1] by default)
+Non-special commit messages must follow one of the style guides ([jQuery Commit Guidelines][1] by default)
+
+### Repo-local install
+
+Commit as usual. Git will trigger commitplease to check your commit message for errors. Invalid messages will prevent the commit, with details about what went wrong and a copy of the input.
+
+### Global install
+
+Navigate to your repository and run the global commitplease executable. By default, it will check all the commit messages. Other examples include (just anything you can pass to `git log` really):
+
+| Use case | command |
+|----------|---------|
+| Check all commits on branch `master` | `commitplease master` |
+| Check the latest 1 commit | `commitplease -1` |
+| Check all commits between `84991d` and `2021ce` | `commitplease 84991d..2021ce` |
+| Check all commits starting with `84991d` | `commitplease 84991d..` |
```

Comment:
I think a common usecase would be to check all commits on a non-master branch, like a feature branch, but ignore any commits that are already on master.